WowPacketParserModule.V8_0_1_27101/Parsers/BattlePetHandler.cs

```diff
@@ -0,0 +1,104 @@
+﻿using WowPacketParser.Enums;
+using WowPacketParser.Misc;
+using WowPacketParser.Parsing;
+
+namespace WowPacketParserModule.V8_0_1_27101.Parsers
+{
+    public static class BattlePetHandler
+    {
+        public static void ReadPetBattleEffectTarget(Packet packet, params object[] idx)
+        {
+            var type = packet.ReadBits("Type", 4, idx); // enum PetBattleEffectTargetEx
```

Comment:
do we have definitions for "PetBattleEffectTargetEx" ?

Reply:
dont think so